tools/clang/test/DXC/denorm.test

```diff
@@ -0,0 +1,5 @@
+// Test for denorm options
+// RUN: %dxc  %S/Inputs/smoke.hlsl /Tps_6_2 /denorm preserve
```

Comment:
These are the only test cases that I can find which pass the `/denorm` flag, and they really only verify that we don't reject the flag.

Is there something trivial that we can check for to verify that option is being interpreted correctly. If not, let's file an issue and reference the issue in a comment here to ensure we follow up.

Reply:
Added https://github.com/microsoft/DirectXShaderCompiler/issues/4930

Reply:
I feel like we can add a trivial check here to verify the `fp32-denorm-mode` attribute is getting generated.